lib/shopify_transporter/pipeline/magento/product/top_level_attributes.rb

```diff
@@ -7,11 +7,21 @@ module Pipeline
     module Magento
       module Product
         class TopLevelAttributes < Pipeline::Stage
+          MAX_NUM_OF_OPTIONS_ACCEPTED = 3
+
           def convert(hash, record)
+            if has_too_many_options?(hash)
+              $stderr.puts "Warning: Product #{hash['product_id']} has too many options."\
+              " Only the first #{MAX_NUM_OF_OPTIONS_ACCEPTED} options will be converted."
+            end
```

Comment:
I suggest moving the whole if block to a private method as well. Maybe `warn_if_too_many_options`?